src/backends/sentry_backend_breakpad.cpp

```diff
@@ -106,6 +113,12 @@ sentry__breakpad_backend_callback(
         if (options->on_crash_func) {
             sentry_ucontext_t *uctx = nullptr;
 
+#if defined(SENTRY_PLATFORM_DARWIN) && !defined(SENTRY_BREAKPAD_SYSTEM)
```

Comment:
Shouldn't we only fill in the data if SENTRY_BREAKPAD_SYSTEM _is_ defined?
```suggestion
#if defined(SENTRY_PLATFORM_DARWIN) && defined(SENTRY_BREAKPAD_SYSTEM)
```

Reply:
No, `SENTRY_BREAKPAD_SYSTEM` means that we are using the system-installed `breakpad`, which is the upstream version and that doesn't provide thread-contexts to the crash handler on macOS.

Reply:
Oh right, yeah now it makes sense!